src/app/templates/partials/report-tab.ftl

```diff
@@ -3,9 +3,26 @@
 <#-- @ftlvariable name="focalArtefactUrl" type="String" -->
 
 <#include "report-title.ftl">
-<#if parameterValues??>
-    <p id="param-values"><span class="text-muted">Parameter values:</span> ${parameterValues}</p>
-</#if>
+<div class="container ml-0 mb-2">
```

Comment:
The container is appearing but it is empty. Did you add name/description values for the examples in the demo?

Reply:
You should see content if you looks at the 'other' report in the `demo` db (not `git`). Most of the other demo reports have no description and no separate display name